lib/rainforest/cli/test_files.rb

```diff
@@ -6,13 +6,16 @@ class RainforestCli::TestFiles
   attr_reader :test_folder, :test_paths, :test_data
 
   def initialize(test_folder = nil)
-    @test_folder = test_folder || DEFAULT_TEST_FOLDER
+    test_folder ||= DEFAULT_TEST_FOLDER
+    # remove trailing slash
+    @test_folder = test_folder.gsub(/#{File::SEPARATOR}$/, '')
```

Comment:
This is a bit confusing (at first glance, it looks like it removes all separators even though it doesn't). I think `File.expand_path` should do what you want, right? (Also avoiding relative path weirdness.)